app/views/notification_mailer/_status.html.haml

```diff
@@ -28,10 +28,10 @@
                               - if status.spoiler_text?
                                 %div.auto-dir
                                   %p
-                                    = Formatter.instance.format_spoiler(status)
+                                    = status.spoiler_text
```

Comment:
`h(status.spoiler_text)`? Also, it's lacking custom emoji rendering.

Reply:
The `h` happens by default unless it's a `html_safe` string.

Reply:
Alright.